app/views/questions/edit.html.slim

```diff
@@ -2,6 +2,8 @@
 
 h1.heading-large
   span.heading-secondary= t('breadcrumb', scope: @form.i18n_scope)
+  span.visuallyhidden
+    | :&nbsp;
```

Comment:
Just to check I'm ready this right, this renders a colon so that screen readers get 
```html
Step x of y : Are you single? 
```
etc?

If so, nice!